edit/edit_test.go

```diff
@@ -62,11 +62,14 @@ var shortenLabelTests = []struct {
 	{"//base:rule", "devtools", "//base:rule"},
 	{"//base:base", "devtools", "//base"},
 	{"//base", "base", ":base"},
+	{"//devtools/buildozer:buildozer", "", "//devtools/buildozer"},
 	{"@r//devtools/buildozer:buildozer", "devtools/buildozer", "@r//devtools/buildozer"},
 	{"@r//devtools/buildozer", "devtools/buildozer", "@r//devtools/buildozer"},
 	{"@r//devtools", "devtools", "@r//devtools"},
 	{"@r:rule", "", "@r:rule"},
 	{"@r", "", "@r"},
+	{"@foo//:foo", "", "@foo"},
+	{"@foo//devtools:foo", "", "@foo"},
```

Comment:
this looks wrong.  while the previous should shorten, here it should remain as @foo//devtools:foo and the package should be devtools